apps/client/src/app/pages/list-details/list-crystals-panel/list-crystals-panel.component.ts

```diff
@@ -12,8 +16,22 @@ export class ListCrystalsPanelComponent {
   @Input()
   crystals: ListRow[] = [];
 
+  constructor(private i18nTools: I18nToolsService, private l12n: LocalizedDataService, private message: NzMessageService, private translate: TranslateService) {};
+
   trackByItem(index: number, item: ListRow): number {
     return item.id;
   }
 
+  public getTextExport(): string {
+    let exportString = "Crystals:\n";
```

Comment:
This needs to be translated, use `${this.translate.instant('Crystals')}` instead.